File: 1353-find-resultant-array-after-removing-anagrams/find-resultant-array-after-removing-anagrams.cpp
class Solution {
public:
    vector<string> removeAnagrams(vector<string>& words) {
        vector<string> ans;
        for (string &word : words) {
            if (ans.empty()) {
                ans.push_back(word);
            } else {
                string prev = ans.back();
                string sortedPrev = prev;
                string sortedCurr = word;
                sort(sortedPrev.begin(), sortedPrev.end());
                sort(sortedCurr.begin(), sortedCurr.end());
                if (sortedPrev != sortedCurr) {
                    ans.push_back(word);
                }
            }
        }
        return ans;
    }
};

File: 1353-find-resultant-array-after-removing-anagrams/find-resultant-array-after-removing-anagrams.go
func sorting(s string) string {
	chars := strings.Split(s, "")
	sort.Strings(chars)
	return strings.Join(chars, "")
    }
func removeAnagrams(words []string) []string {
    var ans[]string
    for _,i := range words{
        if(len(ans)==0){
            ans= append(ans,i);
        }else{
            prev:=sorting(ans[len(ans)-1])
            curr:=sorting(i)
            if(prev!=curr){
                ans=append(ans,i)
            }
        }
    }
    return ans;
}